src/metabase/models/serialization/utils.clj

```diff
@@ -0,0 +1,67 @@
+(ns metabase.models.serialization.utils
```

Comment:
nit: I think IdentityHashable deserved to have its own name rather than `utils`.
How about `metabase.models.serialization.identity`?